lib/include/Network/Clients/MQTT.hpp

```diff
@@ -2,10 +2,10 @@
 #define hpp_CPP_MQTTClient_CPP_hpp
 
 // Configuration is done via macros
-#include "MQTTConfig.hpp"
+#include <Network/Clients/MQTTConfig.hpp>
```

Comment:
This is wrong, IMHO. A file in the same folder must stay with its "" include path.

Reply:
I think there is such a way of thinking. However, since this is a library, it is kinder to write the include path specification from the file in the user-visible path to the user-visible file in a general format (<something/something.hpp>) so that it can be used as a reference when the user includes it by the user.